Python/bytecodes.c

```diff
@@ -4844,6 +4844,10 @@ dummy_func(
             assert(((_PyExecutorObject *)executor)->vm_data.valid);
         }
 
+        tier2 op(_INCREMENT_RUN_COUNT, (--)) {
+            current_executor->run_count++;
```

Comment:
If this becomes a bitfield, we can just store `true` here. That should be a *very* tiny bit faster!

```suggestion
            current_executor->run_count = true;
```